.gitlab/binary_build/cws_injector.yml

```diff
@@ -0,0 +1,34 @@
+---
+.cws_injector-build_common:
+  stage: binary_build
+  needs: ["go_mod_tidy_check"]
+  script:
+    - inv check-go-version
+    - inv -e cws-injector.build
+    - $S3_CP_CMD $CI_PROJECT_DIR/$CWS_INJECTOR_BINARIES_DIR/cws-injector $S3_ARTIFACTS_URI/cws-injector.$ARCH
+
+cws_injector-build_amd64:
+  extends: .cws_injector-build_common
+  rules:
+    !reference [.on_tag_or_a7]
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/deb_x64$DATADOG_AGENT_BUILDIMAGES_SUFFIX:$DATADOG_AGENT_BUILDIMAGES
+  tags: ["runner:main"]
```

Comment:
open question: shouldn't we use k8s runners directly for new jobs?

Reply:
Hey chouquette ! I copy pasted what the `cluster_agent.yaml` used to do ... in August 😅 I've just seen that it was updated to use `tags: ["arch:amd64"]` and `tags: ["arch:arm64"]` respectively, I'll do the same. Thanks for catching this !